src/osemgrep/networking/Semgrep_App.ml

```diff
@@ -9,69 +9,65 @@ open Common
  *)
 
 (*****************************************************************************)
-(* Entry points *)
+(* Types *)
 (*****************************************************************************)
+(* TODO? specify this with atd and have both app + osemgrep use it *)
```

Comment:
YES! We can do that in a separate PR if you prefer, but we must remember to do it ASAP!

Reply:
Def a separate PR, but will add it to my list of tasks